src/bundle/SystemInfo/Collector/IbexaSystemInfoCollector.php

```diff
@@ -272,6 +272,9 @@ private static function getStability(ComposerSystemInfo $composerInfo): string
         return Stability::STABILITIES[$stabilityFlag];
     }
 
+    /**
+     * @param list<string> $packageNames
```

Comment:
Why not array?

Reply:
Actually...

`list<X>` is basically equal to `array<int, X>`, with added assumption that keys will be always incrementing by 1. So it is in fact more narrow than `array`.

But yes, it should not matter in this context.